### ISSUE TITLE
Sileo crashes after jailbreak

### ISSUE THREAD

**Issue by ASCTECyberPatriot** [mineek/Serotonin#94]
I am using an iPhone 13 Pro on version 16.6.1. The jailbreak is successful, and reboot works fine, but opening Sileo or Zebra doesn't work and just crashes. I'm not sure why this is, and I've tried using many versions including latest.

**Comment by aspauldingcode** [comment]
Did you open bootstrap and restart bootstrap?

**Comment by ASCTECyberPatriot** [comment]
Yes. Here's my strategy: Open phone > Bootstrap > Open Serotonin > Jailbreak > Restart userspace > Open Bootstrap > Restart server > Respring

After doing this, it works for mere seconds before the phone freezes, crashes, and reboots. I'm going to try removing all my packages incase they are conflicting.

**Comment by ASCTECyberPatriot** [comment]
Well wouldn't you know it, I charged my phone last night and everything works! How about that. Charge your phone people!
